wp-content/themes/access/gulpfile.babel.js

```diff
@@ -343,19 +342,88 @@ gulp.task('images', callback => {
 /**
  * SVGs
  */
+let list = [];
```

Comment:
@theasteve you can start with items in the `iconsInUse` array here instead of starting with an empty array.